OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -74,11 +74,6 @@ public override Activity Tick(Actor self)
 				self.QueueActivity(mobile.MoveTo(moveTo, 1));
 				self.SetTargetLine(Target.FromCell(self.World, moveTo), Color.Gray, false);
 
-				// TODO: The harvest-deliver-return sequence is a horrible mess of duplicated code and edge-cases
```

Comment:
`moveTo` is just some random free cell in the vincinity around the harvester, though, so `the harvester should move to its assigned/assumed resource field instead of clogging up the base` was awfully implicit if that was the original idea behind this code.